frontend/packages/dev-console/src/utils/pipeline-actions.tsx

```diff
@@ -104,6 +104,12 @@ export const newPipelineRun = (pipeline: Pipeline, latestRun: PipelineRun): Pipe
       trigger: {
         type: 'manual',
       },
+      serviceAccount:
+        latestRun && latestRun.spec && latestRun.spec.serviceAccount
+          ? latestRun.spec.serviceAccount
+          : pipeline && pipeline.spec && pipeline.spec.serviceAccount
```

Comment:
Trying to find where in the docs it says that a `pipeline` has a `serviceAccount` property. I cannot find. Please confirm if this is correct.

Reply:
As there is no creation flow in place for `Pipeline`, if `resources`, `params` and `serviceAccount` is added to the `Pipeline` YAML in `Spec`, all the data should cascade down to every corresponding `PipelineRuns` when `Start Pipeline/rerun Pipeline` is triggered.  
Is there any other option to specifically borrow these properties to pass on to the first-pipelinerun for a pipeline if not present in the Pipeline itself? @siamaksade @vdemeester  @sspeiche